### ISSUE TITLE
Check for different multilabel representations in scikit-learn and adapt auto-sklearn for them

### ISSUE THREAD

**Issue by mfeurer** [automl/auto-sklearn#256]

**Comment by robinsonkwame** [comment]
Can't you just provide a set of multi label capable learners for auto-sklearn to try?

**Comment by mfeurer** [comment]
I'm not sure what you mean here. We're wrapping the learnings in scikit-learn and expose them as being able to handle multilabel classification if they allow it by themselves. What we're not doing at the moment is wrapping other scikit-learn models as multilabel models due to some coding complexity

**Comment by robinsonkwame** [comment]
@mfeurer ah that makes sense, I was referring to being able to use multi label classifiers from sklearn in auto sklearn just by specifying them. I misunderstood the title of this issue, wrapping other sklearn models as multi label models does sound complex.

**Comment by mfeurer** [comment]
Closing this as Auto-sklearn supports multi-label classification and regression. This will only be relevant once scikit-learn supports multioutput-multiclass.
